VirtoCommerce.CatalogModule.Data/Model/ItemEntity.cs

```diff
@@ -344,6 +344,8 @@ public virtual void Patch(ItemEntity target)
             target.ShippingType = this.ShippingType;
             target.Priority = this.Priority;
             target.ParentId = this.ParentId;
+            target.StartDate = this.StartDate;
+            target.EndDate = this.EndDate;
```

Comment:
You forgot to add these fields to the Patch method, without this change, modification the values of this field will not work.

Reply:
this.StartDate and this.EndDate they have already been added